src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -9,6 +9,8 @@
 */
 public class TorpedoStore {
 
+  private Random generator = new Random();
```

Comment:
Én is így csináltam, hiszen így egyetlen Random instance-t hoz létre, máskülönben, ha mindig új Randomot írunk, előfordulhat, hogy nem "true randomot" kapunk (Sonar)